tests/functional/tests/mon/test_mons.py

```diff
@@ -0,0 +1,29 @@
+
+class TestMons(object):
+
+    def test_ceph_mon_package_is_installed(self, CephNode, Package):
+        assert Package("ceph-mon").is_installed
+
+    def test_mon_listens_on_6789(self, CephNode, Socket):
+        assert Socket("tcp://%s:6789" % CephNode["address"]).is_listening
+
+    def test_mon_service_is_running(self, CephNode, Service):
+        service_name = "ceph-mon@ceph-%s" % CephNode["vars"]["inventory_hostname"]
+        assert Service(service_name).is_running
+
+    def test_mon_service_is_enabled(self, CephNode, Service):
+        service_name = "ceph-mon@ceph-%s" % CephNode["vars"]["inventory_hostname"]
+        assert Service(service_name).is_enabled
+
+    def test_can_get_cluster_health(self, CephNode, Command):
+        output = Command.check_output("sudo ceph -s")
```

Comment:
This test was here mainly just to see if ``sudo ceph -s`` would return a proper status code and could connect to the cluster.  Most (maybe all) of our scenarios don't have a setup that would allow for the cluster to reach ``HEALTH_OK``. We might be able to raise the amount of OSDs on the ``centos7|xenial_cluster`` scenarios to achieve that though.

I don't think we should be verifying that *every* testing scenario we have can reach ``HEALTH_OK``.